arangod/Network/NetworkFeature.cpp

```diff
@@ -238,4 +264,59 @@ void NetworkFeature::trackForwardedRequest() {
   ++_forwardedRequests;
 }
 
+std::size_t NetworkFeature::requestsInFlight() const {
+  return _requestsInFlight.load();
+}
+
+bool NetworkFeature::isCongested() const {
+  return _requestsInFlight.load() >= (_maxInFlight * ::CongestionRatio);
+}
+
+bool NetworkFeature::isSaturated() const {
+  return _requestsInFlight.load() >= _maxInFlight;
+}
```

Comment:
All of these use full sequential consistency memory ordering. Can we relax this a bit?